eng/config/PublishData.json

```diff
@@ -200,9 +200,9 @@
         "Shipping",
         "NonShipping"
       ],
-      "vsBranch": "main",
+      "vsBranch": "rel/d17.10",
       "vsMajorVersion": 17,
-      "insertionTitlePrefix": "[d17.10 P1]"
+      "insertionTitlePrefix": "[d17.10]"
```

Comment:
For the time being, if we need to update release/dev17.10  for 17.10 Preview 1, it makes sense to me to keep the "P1".  But, if we don't need to because that's already what the copy of this file in that branch says, this is 100% fine.

Reply:
I don't expect there is a lot traffic next week so will just keep the name align with other release branches